paddle/fluid/imperative/reducer.cc

```diff
@@ -94,14 +94,15 @@ Reducer::Reducer(const std::vector<std::shared_ptr<imperative::VarBase>> &vars,
                  const std::vector<bool> &is_sparse_gradient,
                  std::shared_ptr<imperative::ParallelContext> parallel_ctx,
                  const std::vector<size_t> &group_size_limits,
-                 bool find_unused_vars)
+                 bool find_unused_vars, bool reject_rebuld_group)
```

Comment:
`use_rebuld_group` may be better? And we can make its default value `False`

Reply:
ok， fix it